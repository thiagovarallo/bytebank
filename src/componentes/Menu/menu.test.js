import { render, screen } from "@testing-library/react";
import Menu from ".";

test('Renderizar um link da página inicial', () => {
    render(<Menu />)
    const linkPagInicial = screen.getByText('Inicial')
    expect(linkPagInicial).toBeInTheDocument();
});

test('renderizar uma lista de link', () => {
    render(<Menu />)
    const listLinkMenu = screen.getAllByRole('link');
    expect(listLinkMenu).toHaveLength(4)
})

test('não deve ter no link estrato', () => {
    render(<Menu />)
    const linkEstrato = screen.queryByText('Estrato');
    expect(linkEstrato).not.toBeInTheDocument();
})