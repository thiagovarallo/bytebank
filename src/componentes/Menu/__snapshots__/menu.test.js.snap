// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderizar uma list de link com Class link 1`] = `
Array [
  <a
    class="link"
    href="/"
  >
    Inicial
  </a>,
  <a
    class="link"
    href="/"
  >
    Transferências
  </a>,
  <a
    class="link"
    href="/"
  >
    Investimentos
  </a>,
  <a
    class="link"
    href="/"
  >
    Outros serviços
  </a>,
]
`;
